app/assets/javascripts/oxalis/geometries/materials/node_shader.js

```diff
@@ -247,17 +270,23 @@ void main()
       gl_FragColor  = vec4(1.0);
     };
 
-    // Make active node round
+    // Make active node round and make center a white circle
     if (v_isActiveNode > 0.0) {
-      float r = 0.0, delta = 0.0, alpha = 1.0;
+      float r = 0.0, delta = 0.0, alphaInner = 1.0, alphaOuter = 1.0;
       vec2 cxy = 2.0 * gl_PointCoord - 1.0;
       r = dot(cxy, cxy);
+
       #ifdef GL_OES_standard_derivatives
         delta = fwidth(r);
-        alpha = 1.0 - smoothstep(1.0 - delta, 1.0 + delta, r);
+        alphaOuter = 1.0 - smoothstep(0.0, delta, abs(1.0 - delta - r));
+        alphaInner = 1.0 - smoothstep(1.0 - delta, 1.0 + delta, r / (v_innerPointSize / v_outerPointSize));
+        alphaOuter = max(0.0, alphaOuter - alphaInner);
       #endif
 
-      gl_FragColor = vec4(color, alpha);
+      vec4 inner = vec4(color, alphaInner);
+      // Make the outer ring white
```

Comment:
Outdated, the outer ring has the same color as the inner circle, can probably simply be removed.